app/views/groups/new.html.erb

```diff
@@ -0,0 +1,5 @@
+<section class="container">
+  <h1>New Category</h1>
+  <%= link_to '< Back', groups_path %>
+  <%= render "form", group: @group %>
```

Comment:
[REQUIRED] Kindly include an input field for selecting categories in the new transaction form.
[REQUIRED] Kindly ensure that the user is redirected to the categories page after successful submission. This seems to be the better UX